lib/web/apiserver.go

```diff
@@ -622,9 +622,9 @@ func (h *Handler) bindDefaultEndpoints() {
 	h.POST("/webapi/token", h.WithAuth(h.createTokenHandle))
```

Comment:
You forgot the auth export endpoints on line 618-619

Reply:
Good spot, addressed in 454925b67e5d2364622ded3cd569f38ddd94ba8d